README.md

```diff
@@ -447,6 +447,21 @@ $ REPO_ROOT=`pwd` ginkgo '-focus=sanity' -failFast ./test/e2e/
 ...
 ```
 
+#### Starting test cluster and running E2E tests in DeviceMode:Direct
+
+Export environment variable that guides testing scripts to select deployment files for Direct mode:
+
+```sh
+    $ export PMEM_CSI_DEVICEMODE_DIRECT="-direct"
```

Comment:
I would prefer more generic than making it specific to "direct". Like choosing "device mode" : `PMEM_CSI_DEVICEMODE="lvm"  # for lvm`
     or
`PMEM_CSI_DEVICEMODE="direct" # for direct`

And would prefer to prefix the test variables with TEST_,  how about TEST_DEVICEMODE?